src/app/wallets/update-pending-invoices.ts

```diff
@@ -143,6 +144,9 @@ const updatePendingInvoice = async ({
         paymentHash,
       })
 
+      const eventName = `LNINVOICE-PAYMENT-STATUS-${paymentHash}`
```

Comment:
should we centralize event names in config?

Reply:
Updated